slackbot-background/background.go

```diff
@@ -86,6 +86,17 @@ func InitConfig() {
 		log.Fatalf("Could not create persons api client: %s", err)
 	}
 
+	go func() {
+		for {
+			time.Sleep(time.Hour)
```

Comment:
Is this just an arbitrary value chosen to make sure it's always valid? I was wondering if we should refresh based on failed request and retry or something for example if there are cases where the token could be invalidated that are not temporal, but LGTM.

Reply:
>Is this just an arbitrary value chosen to make sure it's always valid?

Yep. The expiration is much longer than 1hr

>I was wondering if we should refresh based on failed request and retry or something for example if there are cases where the token could be invalidated that are not temporal

This is a good question. Reached out to @gdestuynder about whether that is the case.

Going to merge this as to fix the current expiration bug, but may follow up if there are other cases that are not temporal.

Reply:
i generally refresh on failure, expiry reached, or after 15min, whichever comes first
this means any session lasts at maximum 15min even if the tokens are invalidated for some reason

note that in practice tokens are not revoked, this would be for an incident (operational or security)